docs/_build/.gitignore

```diff
@@ -1 +0,0 @@
-*
```

Comment:
As it turns out these aren't required. I added an ignore for the `docs/_build` dir in the main `.gitignore` file. So I think we're okay here.